src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1168,6 +1191,16 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
             };
         }
 
+        private string TrimTrailingSeparators(string s)
+        {
+            while (s.Length > 0 && SourceFileResolver.IsPathSeparator(s[s.Length - 1]))
+            {
+                s = s.Substring(0, s.Length - 1);
```

Comment:
I don't think this loop can execute more than one iteration unless someone writes something silly on the command-line. This logic is identical to that in PathUtilities. I'm going to leave it.